autoload/kite/client.vim

```diff
@@ -77,6 +78,23 @@ function! kite#client#languages(handler)
 endfunction
 
 
+" Returns max file size in bytes, or -1 if not available.
+function! kite#client#max_file_size()
+  let path = s:max_file_size_path
+  if has('channel')
+    let response = s:internal_http(path, g:kite_short_timeout)
+  else
+    let response = s:external_http(s:base_url.path, g:kite_short_timeout)
+  endif
+  let result = s:parse_response(response)
+  if result.status == 200
+    return result.body / 1024
```

Comment:
The endpoint would return the value in KB, so I think the conversion should actually be `* 1024`

Reply:
That's embarrassing ;)